ibis/backends/tests/test_export.py

```diff
@@ -27,6 +28,7 @@
 
 pd = pytest.importorskip("pandas")
 pa = pytest.importorskip("pyarrow")
+pat = pytest.importorskip("pyarrow.types")
```

Comment:
It would be safe to just do `import pyarrow.types as pat`, right? If that gets us better IDE/typing support, I would advocate for that, even if it doesn't match the `pa = pytest.importorskip("pyarrow")` lines above.

Reply:
but that's a nit, also fine keeping it as is